localgov_step_by_step.info.yml

```diff
@@ -1,8 +1,11 @@
-name: Step by step
+name: LocalGovDrupal Step by step
 type: module
-description: Provides the Step-by-step page.
+description: Provides the Step by step page.
 core_version_requirement: ^8.8 || ^9
 package: LocalGovDrupal
 
 dependencies:
+  - drupal:menu_ui
```

Comment:
It doesn't really depend on menu_ui though? It would make more sense to remove the redundant bit from the configuration?

Reply:
It does indeed depend on menu_ui:

 - https://github.com/localgovdrupal/localgov_step_by_step/blob/a3038815cedd572bd4104f50ee4dd25d8048003d/config/install/node.type.localgov_step_by_step_page.yml#L5
- https://github.com/localgovdrupal/localgov_step_by_step/blob/a3038815cedd572bd4104f50ee4dd25d8048003d/config/install/node.type.localgov_step_by_step_overview.yml#L5

Reply:
Have you been removing it from the node configuration? If so I'll strip it out.

Reply:
If it doesn't _need_ people to edit the menu then I think it should be removed.

Reply:
We haven't discussed menus at all, but I don't think these need to be manually added to menus. We might want to place things in menu for header and footer blocks, but this would be a separate user story and it's not obvious that steps by steps are things that should be added to these.

I'll remove menu_ui config and dependency.